azure-pipelines.yml

```diff
@@ -37,4 +37,4 @@ jobs:
       - bash: echo "##vso[task.prependpath]C:\Users\VssAdministrator\.deno\\bin"
       - bash: npx eslint **/*.ts --max-warnings=0
       - bash: deno.exe run --allow-run --allow-write --allow-read format.ts --check
-      - bash: deno.exe run --allow-run --allow-net --allow-write --allow-read test.ts
+      - bash: deno.exe run --allow-run --allow-net --allow-write --allow-read --config=tsconfig.test.json test.ts
```

Comment:
Why is the --config flag being added here?

Reply:
Well I tried using strict mode in linter but it was not making any difference. So I explicitly tell Deno's TS compiler to use strict mode.